libecho/include/EchoRawConnection.h

```diff
@@ -40,37 +40,26 @@ class EchoRawConnection {
     EchoRawConnection &operator=(EchoRawConnection &&other) noexcept = default;
 
     /**
-     * @brief Sends given byte sequence.
-     *
-     * Anyone can receive it, and there is no guarantee that received transmission will be correct.
-     * @param buffer    Bytes to send.
+     * @brief Starts a message block
      */
-    virtual void send(const std::vector<uint8_t> &buffer) = 0;
+    virtual void sendStart() = 0;
 
     /**
-     * @brief Sends given byte sequence and waits until sending finishes.
+     * @brief Sends given byte sequence
      *
-     * Anyone can receive it, and there is no guarantee that received transmission will be correct.
      * @param buffer    Bytes to send.
      */
-    virtual void sendBlocking(const std::vector<uint8_t> &buffer) = 0;
+    virtual void send(uint8_t *buffer, int size) = 0;
```

Comment:
Mieliśmy przyjmować vector z tego co pamiętam z ustalania, i tak dotychczas było. Dałoby radę to przywrócić?